go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/gballet/go-verkle
 
 go 1.18
 
-require github.com/crate-crypto/go-ipa v0.0.0-20230315201338-1643fdc2ead8
+require github.com/crate-crypto/go-ipa v0.0.0-20230410135559-ce4a96995014
```

Comment:
Basically applying the same change here https://github.com/gballet/go-verkle/pull/339.